src/rcx/src/extRCmodel.cpp

```diff
@@ -876,12 +876,29 @@ extDistWidthRCTable::extDistWidthRCTable(bool over,
   _layerCnt = layerCnt;
   _met = met;
 
+  _firstDiagWidth = nullptr;
```

Comment:
I think it would be a lot less error prone, more readable and concise if all the empty defaults would be using default member initialization (anything that is initialized to `nullptr` here is probably falling into that category).

Members that do get initialized from parameters can also get much more readable by using  the RAII-syntax (`... : _over(over), _layerCnt(layerCnt)...`).

The duplication of code between all these constructors could probably also be reduced by using delegating constructors.